CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 # Unreleased
 
+# 9.11.0 (Feature release)
+
 ## Fixes
 
-- [#938: Update Marked module to fix security issue](https://github.com/alphagov/govuk-prototype-kit/pull/938)
-- [#948: Fix sass files being copied to public directory](https://github.com/alphagov/govuk-prototype-kit/pull/948)
-- [#944: Fix typo that causes ghostMode to be accidentally enabled](https://github.com/alphagov/govuk-prototype-kit/pull/944)
+- [Pull request #938: Update Marked module to fix security issue](https://github.com/alphagov/govuk-prototype-kit/pull/938)
+- [Pull request #944: Fix typo that accidentally enables ghostMode](https://github.com/alphagov/govuk-prototype-kit/pull/944)
```

Comment:
I wonder if there's anything more we could say about this change. Users reading the release notes might not know what ghostMode is but they might recognise the behaviour it alters. This is just a suggestion but maybe something like
> Disable [Browsersync ghostMode](https://browsersync.io/docs/options#option-ghostMode) to stop interactions being mirrored across tabs and devices

Reply:
We could also move it to the top of the list as it potentially has the most impact on users of the kit.